### PR TITLE
Change annotationslegend position

### PR DESCRIPTION
**Pull request by fromdusttilldawn**

### CODE REVIEW

File: views/Choropleth.svelte [hunk 1]
```diff
@@ -74,6 +74,9 @@
           {annotations}
           {annotationRadius} />
       {/if}
+      <AnnotationsLegend
```
Comment (manuelroth) [inline views/Choropleth.svelte:77]: @fromdusttilldawn could you add an` if(mapAnnotations && mapAnnotations.length > 0) `check here. So if no annotations are present, there is no extra margin added?

Screenshot 2021-02-05 at 15 44 02

Comment (fromdusttilldawn) [reply views/Choropleth.svelte:77]: Hm. If there are no annotations, then the Svelte component won't be rendered, right? And if it's not rendered, then the `margin-bottom` property won't have any effect?

Comment (fromdusttilldawn) [reply views/Choropleth.svelte:77]: Ah wait, my bad. I didn't think it through

Comment (manuelroth) [reply views/Choropleth.svelte:77]: Great 👍

**Review by manuelroth — APPROVED** [review]

**Pull request merged**